### PR TITLE
refactor: use new Http library in playground

### PR DESCRIPTION
**Pull request by CaerusKaru**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: modules/playground/src/http/app/http_comp.ts [hunk 1]
```diff
@@ -6,26 +6,27 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
+import {HttpClient} from '@angular/common/http';
 import {Component} from '@angular/core';
-import {Http, Response} from '@angular/http';
-import {map} from 'rxjs/operators';
+
+interface Person {
```
Comment (alexeagle) [inline modules/playground/src/http/app/http_comp.ts]: nit: I'd prefer to make these changes mechanical as possible and keep refactoring in separate PRs, but not a big deal here

**Review by alexeagle — APPROVED** [review]

**Review by IgorMinar — APPROVED** [review]

**Comment by IgorMinar** [comment]
merge-assistance: no g3 impact
